docs/partial_source/deep_dive/ivy_frontends.rst

```diff
@@ -199,18 +205,51 @@ Ivy does not support the unique naming of individual operations, and so we omit
 .. code-block:: python
 
     # in ivy/functional/frontends/tensorflow/math.py
+    @to_ivy_arrays_and_back
     def tan(x, name=None):
         return ivy.tan(x)
 
 Likewise, :code:`tan` is also placed under :mod:`math`.
 By referring to the `tf.math.tan`_ documentation, we add the same arguments, and simply wrap :func:`ivy.tan` in this case.
 Again, we do not support the :code:`name` argument for the reasons outlined above.
 
+**NOTE**
+
+For The :mod:`tf.raw_ops` many functions posses the same behavior as those of :mod:`tf` namespace except that only the functions from :mod:`tf.raw_ops` takes key-word only arguments.
+For this we have a designed decorator to wrap the :mod:`tf` namespace function to satisfy this behavior. Let's take an example of :func:`tf.math.argmax` and :func:`tf.raw_ops.ArgMax`.
```

Comment:
To tackle this disparity in behaviour, the decorator `@map_raw_ops_alias` was designed to wrap these functions that exist in the TensorFlow namespace and reduce the number of implementations necessary.